src/Symfony/Component/Validator/Constraints/Time.php

```diff
@@ -35,16 +35,19 @@ class Time extends Constraint
      */
     protected static $errorNames = self::ERROR_NAMES;
 
+    public $withSeconds = true;
```

Comment:
This could be type as `bool` I guess 🙂

Reply:
If you are talking about consistency in tests, so take a look at constraints. There are no typed properties.

Reply:
Maybe we may add them little by little? Except if there's a reason not to do it which I wouldn't be aware of. That's totally a possibility 😄